controllers/articles.controller.js

```diff
@@ -0,0 +1,21 @@
+const { requireArticles } = require("../models /articles.model");
+const { requireArticleById } = require("../models /articles.model");
+
+exports.getArticles = (req, res) => {
+  requireArticles().then((data) => {
+    res.status(200).send(data);
+  });
+};
+exports.getArticleById = (req, res) => {
+  const { article_id } = req.params;
+
+  if (Number(article_id) !== NaN && Number(article_id) <= 13) {
```

Comment:
This conditional logic is EXTREMELY brittle. This now means that your api will only every serve up to article 13. There are 13 articles in the test data, there are a lot more in the development data which will get hosted.

You should have error handling using PSQL error codes to catch requests that aren't a number, this would be a 22P02 error code.